src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import ApexCharts from 'apexcharts';
+
+(function () {
+  const options = {
+    series: [{
+      name: 'Inflation',
+      data: [2.3, 3.1, 4.0, 10.1, 4.0, 3.6, 3.2, 2.3, 1.4, 0.8, 0.5, 0.2],
```

Comment:
Test data and untranslatable months are alight for test, but I assume this file won't be committed as is.